components/footer/footer.css

```diff
@@ -0,0 +1,20 @@
+.footer {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.footer__title {
+  font-weight: 500;
+  font-size: 38px;
+  line-height: 48px;
+  margin-top: 71px;
```

Comment:
instead of margin-top here, better to use padding on the top level

Reply:
Fixed